src/components/sprint-view-table/tasks-table.tsx

```diff
@@ -69,51 +71,34 @@ const TaskTable = ({project, loggedInpersonId, filter}: Props) => {
           }
           return 0;
         } catch (error) {
-          console.error(`Error fetching time entries for task ${task.id}: ${error}`);
+          const message: string = strings.formatString(strings.sprintRequestError.fetchTasksError, task.id||0, error as string).toString();
+          setError(message);
           return 0;
         }
       }));
       setTimeEntries(fetchedTimeEntries);
+      setLoading(false);
     } catch (error) {
-      console.error(`Error fetching time entries: ${error}`);
-    }
-  };
-
-  /**
-   * Retrieve total time entries for a task
-   */
-  const getTotalTimeEntries = (task: Tasks) => {
-    if (timeEntries.length!==0) {
-      return timeEntries[tasks.indexOf(task)];
+      setError(`${strings.sprintRequestError.fetchTimeEntriesError} ${error}`);
+      setLoading(false);
     }
-    return 0;
-  }
-
-  /**
-   * Lowercase all letters in a string except the first letter except first one
-   */
-
-  const lowerCaseAllWordsExceptFirstLetters = (string: string) => {
-    return string.replace(/\S*/g, (word) => {
-      return word.slice(0, 1) + word.slice(1).toLowerCase();
-    });
   };
 
   return ( 
-    <Card key={0} sx={{ backgroundColor: '#f2f2f2', margin: 0, paddingTop: "5px", paddingBottom: "5px", width: "100%", height: "100", marginBottom: "16px", 
-    '& .high_priority': {
-      color: 'red'}, 
-    '& .low_priority': {
-      color: 'green'} 
+    <Card key={0} sx={{ backgroundColor: "#f2f2f2", margin: 0, paddingTop: "5px", paddingBottom: "5px", width: "100%", height: "100", marginBottom: "16px", 
```

Comment:
Again for all these components fix the indentation, if there are 3 or more props indent each on its own new line